### PR TITLE
phpDoc: read|write only

### PR DESCRIPTION
**Pull request by githubjeka**
Fix #10989
- [x] - phpDoc
- [x] -  build Controller

### CODE REVIEW

**Comment by zinzinday** [comment]
[@property](https://manual.phpdoc.org/HTMLSmartyConverter/PHP/phpDocumentor/tutorial_tags.property.pkg.html)
have you tested for [@yii2-apidoc](https://github.com/yiisoft/yii2-apidoc) ?

**Comment by githubjeka** [comment]
@zinzinday not yet

**Comment by cebe** [comment]
@githubjeka you can just change PhpDocController and re-run it: https://github.com/yiisoft/yii2/blob/master/build/controllers/PhpDocController.php#L686

**Comment by githubjeka** [comment]
@cebe Done

File: framework/behaviors/AttributeTypecastBehavior.php [hunk 1]
```diff
@@ -105,8 +105,6 @@
  * $model->typecastAttributes();
  * ```
  *
- * @property Model|BaseActiveRecord $owner the owner of this behavior.
- *
```
Comment (githubjeka) [inline framework/behaviors/AttributeTypecastBehavior.php]: It has been added to autocompleat by manual . But `php build php-doc/property` remove it.

File: framework/behaviors/AttributeTypecastBehavior.php [hunk 2]
```diff
@@ -105,6 +105,8 @@
  * $model->typecastAttributes();
  * ```
  *
+ * @property Model|BaseActiveRecord $owner the owner of this behavior.
```
Comment (githubjeka) [inline framework/behaviors/AttributeTypecastBehavior.php]: https://github.com/yiisoft/yii2/pull/11834#discussion_r76779456

**Comment by samdark** [comment]
Oh. Seems all PRs for 2.1 were closed automatically...

**Comment by githubjeka** [comment]
Again?

**Pull request closed without merging**